.readthedocs.yaml

```diff
@@ -1,11 +1,18 @@
+# Configuration on how ReadTheDocs (RTD) builds our documentation
+# ref: https://readthedocs.org/projects/2i2c-pilot-hubs/
+# ref: https://docs.readthedocs.io/en/stable/config-file/v2.html
+#
 version: 2
 
-conda:
-  environment: docs/environment.yml
-
 build:
-  image: latest
+  os: ubuntu-20.04
+  tools:
+    # NOTE: This must be mambaforge or it will mistakenly try to invoke `python` to do a conda install
+    # ref: https://stackoverflow.com/questions/69774313/why-does-rtd-run-python-env-create-instead-of-conda-env-create-when-building
+    python: "miniforge-4.10"
```

Comment:
```
Problem in your project's configuration. Invalid "build.tools.python": expected one of (2.7, 3.6, 3.7, 3.8, 3.9, 3.10, pypy3.7, miniconda3-4.7, mambaforge-4.10, 3), got miniforge-4.10
```

Reply:
```suggestion
    python: "miniconda3-4.7"
```

Reply:
```suggestion
    python: "mambaforge-4.10"
```

Reply:
I've seen a transition to mambaforge across many repos. In the dask ecosystem, pangeo, jupyter/docker-stacks, and repo2docker use micromamba now which is more experimental overall I think.

+1 for going for mambaforge over miniforge at this point in general, but it really doesn't matter really.

Reply:
I would prefer to keep miniconda until we have a consolidated story around mamba (which I believe we should eventually migrate).

Reply:
So committing you first option 😉